src/components/VLink.vue

```diff
@@ -1,7 +1,17 @@
 <template>
-  <Component :is="linkComponent" v-bind="linkProperties" v-on="$listeners"
-    ><slot
-  /></Component>
+  <Component
```

Comment:
I'm not sure if it's a good idea to tightly couple the link with the external link icon. This adds styles that might be hard to override in other components. Wouldn't wrapping an external link in another component be a better way of implementing this?

Reply:
My idea behind this was that `VLink` should be the be-all-end-all component for links. Any feature you could want from a link component is supported by it.

In the current implementation it doesn't introduce any style conflicts (_at least not yet_). Do you think it's a good idea to have such a versatile component?